app/src/main/java/woowacourse/shopping/data/datasource/remote/ProductDataSourceImpl.kt

```
@@ -0,0 +1,70 @@
+                        val product =
+                            response.body()?.toDomain()
+                                ?: throw NoSuchElementException("해당 id의 상품을 찾지 못했습니다.")
+                        onResult(product)
+                    }
+                }
+
+                override fun onFailure(
+                    call: Call<ProductContent>,
+                    t: Throwable,
+                ) {
+                }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)